tests/Avalonia.Controls.UnitTests/Platform/DefaultMenuInteractionHandlerTests.cs

```diff
@@ -2,13 +2,38 @@
 using Avalonia.Controls.Platform;
 using Avalonia.Input;
 using Avalonia.Interactivity;
+using Avalonia.VisualTree;
 using Moq;
 using Xunit;
 
 namespace Avalonia.Controls.UnitTests.Platform
 {
     public class DefaultMenuInteractionHandlerTests
     {
+        class FakePointer : IPointer
```

Comment:
Nit: this should be moved to the bottom with the other inner mock classes. Also any reason it doesn't use the `MouseTestHelper` you added?

Reply:
These tests are passing events to some methods instead of properly rising them, so `MouseTestHelper` can not help here.